src/plugins/kibana/public/settings/sections/advanced/advanced_row.html

```diff
@@ -21,13 +21,22 @@
       role="form">
 
       <input
-        ng-show="conf.normal"
+        ng-if="conf.normal"
```

Comment:
I'm all for using `ng-if` - big fan of creating/destroying dom rather than hiding/showing it, but is there a technical reason for it in this PR?